Frontend/library/src/WebRtcPlayer/WebRtcPlayerController.ts

```diff
@@ -1386,7 +1386,12 @@ export class WebRtcPlayerController {
     setUpMouseAndFreezeFrame() {
         // Calculating and normalizing positions depends on the width and height of the player.
         this.videoElementParentClientRect = this.videoPlayer.getVideoParentElement().getBoundingClientRect();
-        this.coordinateConverter.setupNormalizeAndQuantize();
+        const playerElement = this.videoPlayer.getVideoParentElement();
+        const videoElement = this.videoPlayer.getVideoElement();
+        this.coordinateConverter.configure(
+            { width: playerElement.clientWidth, height: playerElement.clientHeight },
+            { width: videoElement.videoWidth, height: videoElement.videoHeight }
+        );
```

Comment:
I think if you were gonna do it like this you also have to bind to element resize events to call configure then too.

Reply:
Sanity check will be testing with page resizing and mouse coordinates still working as expected.

Reply:
This is not really any different from how it was. I just moved the element holding out of the util class so it could be a little more generic. The way it grabs and calculates sizes is still the same.